test/sumo_basic_SUITE.erl

```diff
@@ -139,6 +145,20 @@ delete_module(Module) ->
 
   1 = length(All) - length(NewAll).
 
+do_check_datetime_fields(Module) ->
+  [P0] = sumo:find_by(Module, [{name, <<"A">>}]),
+  P1 = sumo:find(Module, Module:id(P0)),
+  [P2 | _] = sumo:find_all(Module),
+
+  {Date, _} = calendar:universal_time(),
+
+  Date = Module:date(P0),
+  {Date, {_, _, _}} = Module:datetime(P0),
+  Date = Module:date(P1),
+  {Date, {_, _, _}} = Module:datetime(P1),
+  Date = Module:date(P2),
+  {Date, {_, _, _}} = Module:datetime(P2).
+
```

Comment:
You should also test that setting a value (e.g. `Module:datetime(P2, {datetime, …})`) works.